packages/st2/debian/st2.st2auth.init

```diff
@@ -71,6 +71,8 @@ do_start()
   start-stop-daemon --start --quiet --chuid $RUNAS_USER:$RUNAS_GROUP --umask $UMASK --background --pidfile $PIDFILE \
      --exec $DAEMON -- $DAEMON_ARGS \
     || return 2
+  # wait for st2auth readiness to serve the requests
+  timeout 5 bash -c 'until echo > /dev/tcp/127.0.0.1/9100; do sleep 0.5; done' || return 2
```

Comment:
From the https://stackstorm.slack.com/archives/C029K4CBU/p1489841065872703 it looks like `st2-auth-backend-etenrprise` what broke the things.

Looks like resolved.